src/Components/Directory.Build.props

```diff
@@ -1,14 +1,6 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
-  <PropertyGroup>
-    <!-- Workaround for https://github.com/dotnet/aspnetcore/issues/5486 which requires the bin and obj directory be in the project directory -->
```

Comment:
This issue was resolved. Having this was causing issues in where we discover the SDK outputs (since it's relative to the project directory rather than the ArtifactsDir)